StyleCop.Analyzers/StyleCop.Analyzers/ReadabilityRules/SA1137ElementsShouldHaveTheSameIndentation.cs

```diff
@@ -526,9 +526,22 @@ private static void CheckBraces(SyntaxNodeAnalysisContext context, SyntaxToken o
             SyntaxTrivia closeBraceIndentationTrivia = closeBraceToken.LeadingTrivia.LastOrDefault();
             string closeBraceIndentation = closeBraceIndentationTrivia.IsKind(SyntaxKind.WhitespaceTrivia) ? closeBraceIndentationTrivia.ToString() : string.Empty;
 
-            if (!string.Equals(openBraceIndentation, closeBraceIndentation, StringComparison.Ordinal))
+            if (closeBraceToken.IsOnlyPrecededByWhitespaceInLine())
             {
-                ReportDiagnostic(context, closeBraceToken, closeBraceIndentationTrivia, closeBraceIndentation, openBraceIndentation);
+                if (!string.Equals(openBraceIndentation, closeBraceIndentation, StringComparison.Ordinal))
+                {
+                    ReportDiagnostic(context, closeBraceToken, closeBraceIndentationTrivia, closeBraceIndentation, openBraceIndentation);
+                }
+            }
+            else
+            {
+                // In this case, the closing brace does not seem to have any leading trivia,
+                // but the previous token can have trailing trivia.
+                SyntaxTrivia prevTokenTrailingTrivia = closeBraceToken.GetPreviousToken().TrailingTrivia.FirstOrDefault();
+                string prevTokenTrailingWhitespace = prevTokenTrailingTrivia.IsKind(SyntaxKind.WhitespaceTrivia) ? prevTokenTrailingTrivia.ToString() : string.Empty;
+                string expectedCloseBraceIndentation = Environment.NewLine + openBraceIndentation;
+
+                ReportDiagnostic(context, closeBraceToken, prevTokenTrailingTrivia, prevTokenTrailingWhitespace, expectedCloseBraceIndentation);
```

Comment:
It seems like SA1137 should not be reported here at all. We already get SA1500 here, and that has the correct fix.

Reply:
I see what you mean. Didn't think of that. Alright, so I'm closing this PR.